test/shaping/data/in-house/tests/variations-space.tests

```diff
@@ -0,0 +1,2 @@
+../fonts/ab40c89624a6104e5d0a2308e448a989302f515b.ttf:--variations=wdth=60:U+0020:[space=0+266]
```

Comment:
Just curious, are these implicitly run with `--font-funcs=ot`?

Thanks for the quick fix!

Reply:
It tests both https://github.com/harfbuzz/harfbuzz/blob/ba810ce/test/shaping/run-tests.py#L108-L109

Reply:
> Thanks for the quick fix!

No problem, great that ot funcs are being tested somewhere more seriously :) wishing adoption of more parts of it also